userspace/libsinsp/examples/util.cpp

```diff
@@ -52,288 +52,9 @@ std::string get_event_category(ppm_event_category category)
-        case PPME_SYSCALL_SEMCTL_E:
-        case PPME_SYSCALL_SEMCTL_X: return "semctl";
-        case PPME_SYSCALL_SEMGET_E:
-        case PPME_SYSCALL_SEMGET_X: return "semget";
-        case PPME_SYSCALL_SEMOP_E:
-        case PPME_SYSCALL_SEMOP_X: return "semop";
-        case PPME_SYSCALL_SETNS_E:
-        case PPME_SYSCALL_SETNS_X: return "setns";
-        case PPME_SYSCALL_SETRESGID_E:
-        case PPME_SYSCALL_SETRESGID_X: return "setresgid";
-        case PPME_SYSCALL_SETRESUID_E:
-        case PPME_SYSCALL_SETRESUID_X: return "setresuid";
-        case PPME_SYSCALL_SETSID_E:
-        case PPME_SYSCALL_SETSID_X: return "setsid";
-        case PPME_SYSCALL_TGKILL_E:
-        case PPME_SYSCALL_TGKILL_X: return "tgkill";
-        case PPME_SYSCALL_TKILL_E:
-        case PPME_SYSCALL_TKILL_X: return "tkill";
-        case PPME_SYSCALL_UNSHARE_E:
-        case PPME_SYSCALL_UNSHARE_X: return "unshare";
-        case PPME_SYSCALL_VFORK_E:
-        case PPME_SYSCALL_VFORK_20_E:
-        case PPME_SYSCALL_VFORK_X:
-        case PPME_SYSCALL_VFORK_20_X: return "vfork";
-
-        //
-        // Socket syscalls
-        // 
-        case PPME_SOCKET_SOCKET_E:
-        case PPME_SOCKET_SOCKET_X: return "socket";
-        case PPME_SOCKET_BIND_E:
-        case PPME_SOCKET_BIND_X: return "bind";
-        case PPME_SOCKET_CONNECT_E:
-        case PPME_SOCKET_CONNECT_X: return "connect";
-        case PPME_SOCKET_LISTEN_E:
-        case PPME_SOCKET_LISTEN_X: return "listen";
-        case PPME_SOCKET_ACCEPT_5_E:
-        case PPME_SOCKET_ACCEPT_5_X: return "accept";
-        case PPME_SOCKET_GETSOCKNAME_E:
-        case PPME_SOCKET_GETSOCKNAME_X: return "getsockname";
-        case PPME_SOCKET_GETPEERNAME_E:
-        case PPME_SOCKET_GETPEERNAME_X: return "getpeername";
-        case PPME_SOCKET_GETSOCKOPT_E:
-        case PPME_SOCKET_GETSOCKOPT_X: return "getsockopt";
-        case PPME_SOCKET_SOCKETPAIR_E:
-        case PPME_SOCKET_SOCKETPAIR_X: return "socketpair";
-        case PPME_SOCKET_SENDTO_E:
-        case PPME_SOCKET_SENDTO_X: return "sendto";
-        case PPME_SOCKET_RECVFROM_E:
-        case PPME_SOCKET_RECVFROM_X: return "recvfrom";
-        case PPME_SOCKET_SHUTDOWN_E:
-        case PPME_SOCKET_SHUTDOWN_X: return "shutdown";
-        case PPME_SOCKET_SETSOCKOPT_E:
-        case PPME_SOCKET_SETSOCKOPT_X: return "setsocktopt";
-        case PPME_SOCKET_SENDMSG_E:
-        case PPME_SOCKET_SENDMSG_X: return "sendmsg";
-        case PPME_SOCKET_ACCEPT4_5_E:
-        case PPME_SOCKET_ACCEPT4_5_X: return "accept";
-        case PPME_SOCKET_SENDMMSG_E:
-        case PPME_SOCKET_SENDMMSG_X: return "sendmsg";
-        case PPME_SOCKET_RECVMSG_E:
-        case PPME_SOCKET_RECVMSG_X: return "recvmsg";
-        case PPME_SOCKET_RECVMMSG_E:
-        case PPME_SOCKET_RECVMMSG_X: return "recvmmsg";
-        default: return "UNKNOWN " + to_string(type);
-    };
+	if (type > 1 && type < PPM_EVENT_MAX)
```

Comment:
Just out of curiosity, is this `type >= 1` meant to exclude the `PPME_GENERIC_E` and `PPME_GENERIC_X` types? Maybe we can change the condition to be a little more explicit about this?
```suggestion
	if (type < PPM_EVENT_MAX && type != PPME_GENERIC_E && type != PPME_GENERIC_X)
```